misc/ng-docs.css

```diff
@@ -44,6 +44,12 @@
   margin-right: 0px;
 }
 
+a.navbar-brand img {
+  height: inherit;
+  width: inherit;
+  margin-top: inherit;
+}
+
```

Comment:
Wondering if this has any adverse effects on other examples. Did you look at the vertical navigation example to verify it does not?

Reply:
I checked vertical nav and it looks to be fine.